source/extensions/filters/listener/proxy_protocol/proxy_protocol.cc

```diff
@@ -299,49 +299,43 @@ bool Filter::parseExtensions(os_fd_t fd, uint8_t* buf, size_t buf_size, size_t*
  *        See https://www.haproxy.org/download/2.1/doc/proxy-protocol.txt for details
  */
 void Filter::parseTlvs(const std::vector<uint8_t>& tlvs) {
-  size_t size = tlvs.size();
-  uint8_t tlv_type;
-  uint8_t tlv_length_upper;
-  uint8_t tlv_length_lower;
-  size_t tlv_value_length;
   size_t idx{0};
-  while (idx < size) {
-    // get type
-    tlv_type = tlvs[idx];
+  while (idx < tlvs.size()) {
+    // Get TLV type.
```

Comment:
Delete this comment; the code is self-documenting about what it is doing

Reply:
removed